packages/headless/src/api/commerce/product-listings/v2/product-listing-v2-params.ts

```diff
@@ -0,0 +1,92 @@
+import {FacetValueRequest} from '../../../../features/facets/facet-set/interfaces/request';
+import {AnyFacetRequest} from '../../../../features/facets/generic/interfaces/generic-facet-request';
+import {RangeValueRequest} from '../../../../features/facets/range-facets/generic/interfaces/range-facet';
+import {SortCriterion} from '../../../../features/sort/sort';
+import {
+  HTTPContentType,
+  HttpMethods,
+  PlatformClientCallOptions,
+} from '../../../platform-client';
+
+export enum Mode {
+  Live = 'live',
+  Sample = 'Sample',
```

Comment:
The API will accept lowercased modes
```suggestion
  Sample = 'sample',
```